### PR TITLE
DDO-945 First pass at cromiam Helm chart

### PR DESCRIPTION
**Pull request by choover-broad**
Initial cromiam Helm chart implementation

### CODE REVIEW

**Review by emandakh — APPROVED** [review]

File: charts/cromiam/templates/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-cm
```
Comment (mflinn-broad) [inline charts/cromiam/templates/configmap.yaml:4]: I noticed Greg was using a  `{{ .Values.name }}` for this purpose and I think that is a good pattern. It makes it much easier if we need to make tweaks to application names or override them for some reason down the road.

File: charts/cromiam/templates/ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+metadata:
+  name: {{ .Chart.Name }}-ingress
+  labels:
+{{ include "cromiam.labels" . | indent 4 }}
+  annotations:
+    networking.gke.io/v1beta1.FrontendConfig: "{{ .Chart.Name }}-ingress-frontendconfig"
+    kubernetes.io/ingress.global-static-ip-name: {{ required "ingress.staticIpName value is required" .Values.ingress.staticIpName | quote }}
+{{- if not (empty .Values.ingress.preSharedCerts) }}
+    ingress.gcp.kubernetes.io/pre-shared-cert: {{ .Values.ingress.preSharedCerts | join "," | quote }}
+{{- end }}
```
Comment (mflinn-broad) [inline charts/cromiam/templates/ingress.yaml:13]: If you aren't including the logic to pull the  certs from vault also then I don't think this conditional is necessary. Not a big deal though 😄

**Review by mflinn-broad — APPROVED** [review]
woohoo!

File: charts/cromiam/templates/ingress.yaml (hunk 2 above)
Comment (choover-broad) [reply charts/cromiam/templates/ingress.yaml:13]: Don't want to supply an empty string to the annotation and get potentially undefined behavior 🤷‍♀️

Comment (mflinn-broad) [reply charts/cromiam/templates/ingress.yaml:13]: not trying to bike shed but would it be better to use the `required` function with the value in that case?

File: charts/cromiam/templates/configmap.yaml (hunk 1 above)
Comment (choover-broad) [reply charts/cromiam/templates/configmap.yaml:4]: Is the idea that anywhere .Chart.Name would be used, `.Values.name` is substituted? I.e., I shouldn't have any uses of `.Chart.Name` in the chart at all?

Comment (mflinn-broad) [reply charts/cromiam/templates/configmap.yaml:4]: I think so @gmalkov any thoughts on this?

Comment (mflinn-broad) [reply charts/cromiam/templates/configmap.yaml:4]: Also, didn't mean for these to be blockers to you merging the pr. I intended them more as general best practice questions 😄

**Pull request merged**